README.md

```diff
@@ -475,48 +477,67 @@ curl -v \
-        \"conversationParticipants\": [
-            {
-                \"displayName\": \"<displayName>\",
-                \"participant\": {
-                    \"subject\": \"<participantSubject>\",
-                    \"role\": \"<ParticipantRole>\",
-                    \"appType\": \"<AppType>\"
-                },
-                \"joinedTime\": <Timestamp in Unix Epoch Milliseconds>
-            }
-        ],
-        \"conversationEntries\": [
-            {
-                \"clientTimestamp\": <Timestamp in Unix Epoch Milliseconds>,
-                \"sender\": {
-                    \"subject\": \"<senderSubject>\",
-                    \"role\": \"<ParticipantRole>\",
-                    \"appType\": \"<AppType>\"
-                },
-                \"entryPayload\": {
-                    \"id\": \"<messageId>\",
-                    \"attachmentIndex\": <attachmentIndex>,
-                    \"contentLength\": <contentLength>,
-                    \"text\": \"Hi There\"
+curl -v \
+-H "Authorization: Bearer <AccessToken>" \
+-H "content-type: application/json" \
+-H "OrgId: <OrgId>" \
+-H "AuthorizationContext: <AuthorizationContext>" \
+-H "AuthorizationContextType: <AuthorizationContextType>" \
+-H "RequestId: <RequestId>" \
+-X POST -d "{
+  \"channelAddressIdentifier\": \"<channelAddressIdentifier>\",
+  \"conversationParticipants\": [
+    {
+      \"displayName\": \"<displayName>\",
+      \"participant\": {
+            \"subject\": \"<unique identifier for the participant>\",
+            \"role\": \"<ParticipantRole>\",
+            \"appType\": \"<AppType>\"
+        },
+      \"joinedTime\":  <Timestamp in Unix Epoch Milliseconds>
+    }
+  ],
+  \"conversationEntries\": [
+    {
+        \"clientTimestamp\":  <Timestamp in Unix Epoch Milliseconds>,
+        \"sender\": {
+            \"subject\": \"<unique identifier for the participant>\",
+            \"role\": \"<ParticipantRole>\",
+            \"appType\": \"<AppType>\"
+        },
+        \"entryPayload\": {
+            \"entryType\": \"Message\",
+            \"id\": \"<payloadId>\",
+            \"abstractMessage\": {
+                \"messageType\": \"StaticContentMessage\",
+                \"inReplyToMessageId\": <inReplyToMessageId>,
+                \"id\": \"<messageId>\",
+                 \"references\": [{
+                    \"recordId\": \"068LT000000DoS1\",
+                    \"id\": \"1427e833-9427-4947-88ea-6c029e4e357f\"
+                  }],
+                \"staticContent\": {
+                    \"formatType\": \"Attachments\",
+                    \"text\": <text>,
+                    \"attachments\": [
+                      {
+                        \"name\": \"pdf-sample.pdf\",
+                        \"attachmentUploadResult\": <attachmentUploadResult>,
+                        \"id\": \"<attachmentId>\",
+                        \"mimeType\": \"application/pdf\",
+                        \"url\": \"<attachmentUrl>\",
```

Comment:
mention that this is the public link of the uploaded file.

Reply:
updated